drivers/docker/driver_test.go

```diff
@@ -1553,6 +1553,48 @@ func TestDockerDriver_CPUSetCPUs(t *testing.T) {
 	}
 }
 
+func TestDockerDriver_CPUSetMEMs(t *testing.T) {
+	ci.Parallel(t)
+	testutil.DockerCompatible(t)
+	testutil.CgroupsCompatible(t)
+
+	testCases := []struct {
```

Comment:
Thanks! LGTM

nit:  maybe add a test case of invalid  `CPUSetMems` ?

Reply:
Added `TestDockerDriver_InvalidCPUSetMEMs`